src/webassets/bundle.py

```diff
@@ -741,7 +744,7 @@ def _urls(self, ctx, extra_filters, *args, **kwargs):
             # With ``auto_build``, build the bundle to make sure the output is
             # up to date; otherwise, we just assume the file already exists.
             # (not wasting any IO ops)
-            if ctx.auto_build:
+            if ctx.auto_build or self.auto_build:
```

Comment:
``ctx.auto_build`` should already consider ``self._config['auto_build']``. This is not the case in your experience?